custom/enikshay/case_utils.py

```diff
@@ -298,7 +298,7 @@ def update_case(domain, case_id, updated_properties, external_id=None,
     )
 
 
-def get_person_locations(person_case):
+def get_person_locations(person_case, episode_case=None):
```

Comment:
It's a bit confusing that the behaviour of this method is different depending on if you pass in just the person or both. Might be worth adding a docstring here.

Reply:
or make the episode case mandatory

Reply:
Cant make it mandatory since we don't need this change for other place like private locations and is needed to be absent when not needed, plus then would need to make changes for the calling method to tell wether we should pick owner_id or d_f_id. 
I think its a fine way of saying use a if passed else use b.
Will add docstring